### DISCUSSION TITLE
We need to create another repository named `create-rome-app` to do the initial things.

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [rome/tools#1237] [General]
We should create an official scaffolding tool for Rome.  We can create another repository named `create-rome-app` to do the initial things. Because the developer was used to use such scaffolding as `vue-cli` and `create-react-app`. I believe that Rome will replace Webapck, Pretieer, Eslint, Babel, and so on. So  It's vital for us to create an official scaffolding. It will more efficient for developers to use Rome with frameworks. Because there are so many different frameworks. such as Vue, Svelte,   React, Angular, Svelte, Preact, and original Html. It's such a big project that we should create another repository. I want to join the Rome org to do these things. Let's make Rome better together. If I joined Rome org. I will do these things besides buying the `create-rome-app` domain and building the same site for developers to use. Think twice, please.  Thank you.

@bitpshr @sebmck @diokey @EduardoLopes @ematipico @yassere 

Are there only I need to mention? If I left anyone please mention each other. thank you.

**Comment by ematipico** [comment]
Hi @iam-frankqiu , what do you expect from such command? We need to understand first that Rome is a **tool** and not a framework. Rome should build the foundations of the infrastructure of the project, not prepare a project (react, svelte, etc.).

That being said, I would suggest to take a step back and first thinking about the requirements of what are the potential features of this discussion.

**Reply by iam-frankqiu** [reply]
> That being said, I would suggest to take a step back and first thinking about the requirements of what are the potential features of this discussion.

Sorry. Maybe I have made a misunderstanding. I  know it's a tool. Rome is pretty fine now. I want to make Rome better.  What I suggest is that it will be better if Rome has scaffolding. Some developers like me were used to use this scaffolding like `create-react-app`. It will more convenient for developers to start a web app. It will also make Rome become more popular I think. I hope I have the pleasure to do such a thing for Rome. That's it. Thank you.
